apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/index.tsx

```diff
@@ -11,8 +11,32 @@ declare global {
 	}
 }
 
+const createLocalizedDescriptionWithLearnMore = (
+	title: string,
+	description: string | ReactElement< string | JSXElementConstructor< any > >,
+	url: string
+) => {
+	const localizedUrl = localizeUrl( url, window.wpcomBlockDescriptionLinksLocale );
+
+	return createInterpolateElement( '<InlineSupportLink />', {
+		InlineSupportLink: (
+			<DescriptionSupportLink title={ String( title ) } url={ localizedUrl }>
+				{ description }
+			</DescriptionSupportLink>
+		),
+	} );
+};
+
+const processedBlocks: { [ key: string ]: true } = {};
```

Comment:
Observation, don't care if you address this. The values of this object are always true, it could've just been an array 🤔

Reply:
Using an arrray would have a  O(n) complexity to search for a given element. with an object, we'll always have O(1).